src/lib/links.store.ts

```diff
@@ -0,0 +1,14 @@
+import { readable, type Readable } from 'svelte/store';
+import { z } from 'zod';
```

Comment:
As requested @cerebrl here is the links store re-written using Zod. This actually seemed quite simple to do. Let me know what you think.